src/pages/rest/b2b/roles.md

```diff
@@ -0,0 +1,1214 @@
+---
+title: Manage company roles
+description: Create and assign user roles for B2B users
+ee_only: True
+---
+
+import * as Vars from '../../../data/vars.js';
+
+# Manage company roles
+
+Within a company, customers may have different job roles, levels of responsibility, and access to information about their company. <Vars.sitedatavarb2b/> defines several types of system resources, and the Company Admin (or an integration that operates on the behalf of the Company Admin) grants or denies access to these resources using company roles. The Company Admin has access to all resources.
+
+<Vars.sitedatavarb2b/> defines the following types of resources:
+
+-  Sales
+-  Purchase Orders
+-  Negotiable quotes
+-  Company profile
+-  Company user management
+-  Company credit
+
+Each of these resources contains a hierarchy of other resources. When a Company Admin grants or blocks access to a resource from the store UI, the action applies to all sub-resources, unless explicitly overridden. However, if you grant or block access using web APIs, you must specify each resource individually.
+
+The following table lists all the resources that are available to the customers defined with a company. To visualize the resource hierarchy, log in to a store as the Company Admin and select **Roles and Permissions**, then click the **Edit** action next to the Default User role.
+
+<p></p>
+
+<details>
```

Comment:
I have a personal bias against collapsible elements like this because if you attempt to search on a page you have to expand the data set to do a thorough search. Feel free to leave it, but I recommend using collapsible elements judiciously.